openmc/mesh.py

```diff
@@ -611,6 +612,8 @@ class UnstructuredMesh(MeshBase):
     ----------
     filename : str
         Location of the unstructured mesh file
+    length_multiplier: float
```

Comment:
Let's move this to the end of the list here so the order of the parameters matches the signature of the class constructor.

Also, if you wouldn't mind removing the `size` parameter docstring while we're here it would be much appreciated. I must've left that in by mistake.